winml/lib/Api/LearningModel.cpp

```diff
@@ -5,11 +5,11 @@
 
 #include "LearningModel.h"
 
+#ifdef USE_DML
```

Comment:
The WinML side of this needs to keep using the MLOperatorauthorImpl in order to do custom ops.

this is all cleanly layered behind the adapter.  can we leave just the winml side still plumbed - and take a TODO for the ort + adapter side?

Reply:
ok. sounds good.